salt/data-pipeline/bigquery-views.sls

```diff
@@ -9,7 +9,7 @@ bigquery-view docker image:
 re-materialise views daily:
     cron.present:
         - user: {{ pillar.elife.deploy_user.username }}
-        - name: docker run --rm -e DATA_PIPELINE_BQ_PROJECT=elife-data-pipeline elifesciences/data-pipeline-bigquery-views:{{ pillar.data_pipeline.bigquery_views.revision }} ./views-cli.sh --dataset={{ pillar.elife.env }} materialize-views
+        - name: docker run --rm -v /srv/nifi/conf/gcs.json:/root/.config/gcs.json -e GOOGLE_APPLICATION_CREDENTIALS=/root/.config/gcs.json -e DATA_PIPELINE_BQ_PROJECT=elife-data-pipeline elifesciences/data-pipeline-bigquery-views:{{ pillar.data_pipeline.bigquery_views.revision }} ./views-cli.sh --dataset={{ pillar.elife.env }} materialize-views
```

Comment:
Where do this come from on the host? The formula too?

Reply:
dummy version here:
https://github.com/elifesciences/data-pipeline-formula/blob/master/salt/data-pipeline/config/srv-nifi-conf-gcs.json

private version here:
https://github.com/elifesciences/builder-private/blob/master/salt/data-pipeline/config/srv-nifi-conf-gcs.json

this change will suffice for now but I should move them out of NiFi to somewhere more obvious. The google cloud tools don't pick up on default locations of the credentials like the AWS tools do and must be specified with an envvar (like in the change above) but I don't see why they couldn't live alongside each other for convenience